client/src/partials/debtors/groups.create.html

```diff
@@ -0,0 +1,225 @@
+            ng-model="GroupEditCtrl.group.account_id"
+            autocomplete="off"
+            autocorrect="off"
+            autocapitalize="off"
+            required>
+
+          <div class="help-block" ng-messages="debtorGroup.account_id.$error" ng-show="debtorGroup.$submitted">
+            <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+          </div>
+        </div>
+
+        <!-- @todo The UI should show an option that allows you apply a price list, if this is checked then you can select an option -->
+        <div
+          class="form-group has-feedback"
+          ng-class="{'has-error' : debtorGroup.price_list.$invalid && debtorGroup.$submitted}">
+
+          <label class="control-label">{{ "TABLE.COLUMNS.PRICE_LIST" | translate }}</label>
+          <select
+            name="price_list"
+            ng-model="GroupEditCtrl.group.price_list_uuid"
+            ng-options="priceList.uuid as priceList.label for priceList in GroupEditCtrl.priceLists"
+            class="form-control">
+            <option value="">{{ "PRICE_LIST.NONE" | translate }}</option>
+          </select>
+
+          <div class="help-block" ng-messages="debtorGroup.price_list.$error" ng-show="debtorGroup.$submitted">
+            <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+          </div>
+        </div>
+
+        <div
+          class="form-group has-feedback"
+          ng-class="{'has-error' : debtorGroup.max_credit.$invalid && debtorGroup.$submitted}">
+
+          <label class="control-label">{{ "FORM.LABELS.MAX_CREDIT" | translate }}</label>
+          <p class="text-info"><span class="glyphicon glyphicon-info-sign"></span> {{ "FORM.LABELS.MAX_CREDIT_INFO" | translate }}</p>
+          <input type="number" ng-model="GroupEditCtrl.group.max_credit" name="max_credit" class="form-control" autocomplete="off" bh-integer required>
+
+          <div class="help-block" ng-messages="debtorGroup.max_credit.$error" ng-show="debtorGroup.$submitted">
+            <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+          </div>
+        </div>
+
+        <div class="form-group">
+          <label class="control-label">{{ "FORM.LABELS.NOTES" | translate }}</label>
+          <textarea class="form-control" ng-model="GroupEditCtrl.group.note" autocomplete="off" name="notes"></textarea>
+        </div>
+
+        <div class="form-group">
+          <label class="control-label">{{ "FORM.LABELS.PHONE" | translate }}</label>
+          <input name="phone" ng-model="GroupEditCtrl.group.phone" autocomplete="off" class="form-control">
+        </div>
+
+        <div
+          class="form-group"
+          ng-class="{'has-error' : debtorGroup.email.$invalid && debtorGroup.$submitted}">
+          <label class="control-label">{{ "FORM.LABELS.EMAIL" | translate }}</label>
+          <input name="email" ng-model="GroupEditCtrl.group.email" type="email" autocomplete="off" class="form-control">
```

Comment:
For future reference, you can set `autocomplete="off"`, `autocapitalize="off"`, and `autocorrect="off"` on the top-most `<form>` element.  It will turn all these off for every form input.  See #340 for more details.